ci/get_labels/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+var prUrl = flag.String("url", "", "Url of pull request")
+
+func main() {
+	flag.Parse()
+	labelsUrl := strings.Replace(*prUrl, "/pulls/", "/issues/", 1)
+	labelsUrl += "/labels"
+	resp, err := http.Get(labelsUrl)
+	if err != nil {
+		panic(err)
+	}
+	defer resp.Body.Close()
+
+	type Labels struct {
+		Name string
+	}
+
+	var responseLabels []Labels
+	err = json.NewDecoder(resp.Body).Decode(&responseLabels)
+	if err != nil {
+		panic(err)
+	}
+
+	var output string
+	for _, label := range responseLabels {
+		output += label.Name + " "
```

Comment:
Up to you, but it might be nicer here instead to just `fmt.Printf("%s ", label.Name)` instead of using an output variable and printing at the end.

Reply:
Changed in: [8d1c90b](https://github.com/GoogleCloudPlatform/ops-agent/pull/818/commits/8d1c90b269b5dd22a9b4ee640a6a51609a31c97f)